.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "plate-tectonics"]
+	path = plate-tectonics
+	url = https://github.com/Mindwerks/plate-tectonics.git
```

Comment:
We doing submodules now?  I've always had painful experiences here. Is this for appveyer?